src/posthog-persistence.ts

```diff
@@ -23,6 +23,8 @@ export const SESSION_RECORDING_ENABLED_SERVER_SIDE = '$session_recording_enabled
 export const CONSOLE_LOG_RECORDING_ENABLED_SERVER_SIDE = '$console_log_recording_enabled_server_side'
 export const SESSION_ID = '$sesid'
 export const ENABLED_FEATURE_FLAGS = '$enabled_feature_flags'
+const USER_STATE = '$user_state'
```

Comment:
Any strong reason to break the `export` pattern? 🤔

Reply:
It's totally internal to this class... I only have it as a variable to since it's in three places in the file.